content/docs/dev/develop/clients/introduction.md

```diff
@@ -0,0 +1,30 @@
+---
+title: TiKV Clients
+description: Interact with TiKV using the raw key-value API or the transactional key-value API.
+menu:
+    "dev":
+        parent: Develop-dev
+        weight: 1
+        identifier: TiKV Clients-dev
+---
+
+TiKV offers two APIs that you can interact with:
+
+| API           | Description                                                                           | Atomicity     | Usage scenarios                                                                      |
+|:------------- |:------------------------------------------------------------------------------------- |:------------- |:-------------------------------------------------------------------------------- |
+| Raw           | A lower-level key-value API to interact directly with individual key-value pairs  | Single key    | Your application requires low latency and does not involve distributed transactions. |
+| Transactional | A higher-level key-value API to provide ACID semantics.                            | Multiple keys | Your application requires distributed transactions.                              |
+
+{{< warning >}}
+It is **not supported** to use both the raw and transactional APIs on the same keyspace.
```

Comment:
This warning can be changed as apiv2 can support both raw & txn apis.

Reply:
Fixed, PTAL~